wof/examples/flask/odm2/timeseries/odm2_timeseries_dao.py

```diff
@@ -370,3 +412,13 @@ def get_qualcontrollvls_by_ids(self, qual_control_lvl_id_arr):
             w_pl = model.QualityControlLevel(pl[i])
             pl_arr.append(w_pl)
         return pl_arr
+
+
+def _get_tsrv_enddatetimes(db_session, resultids):
+    edt_dict = dict(db_session.query(odm2_models.TimeSeriesResultValues.ResultID,
```

Comment:
Add to docstring: ":param resultids: List of ResultID values (even if it's a single value)"